src/ui/restaurantListRenderer.ts

```diff
@@ -0,0 +1,26 @@
+import { RestaurantType } from '../type';
+import { renderRestaurant } from '../templates';
+import { $ } from '../util/selector';
+import '../assets/category-asian.png';
+import '../assets/category-chinese.png';
+import '../assets/category-etc.png';
+import '../assets/category-japanese.png';
+import '../assets/category-korean.png';
+import '../assets/category-western.png';
+import '../assets/favorite-icon-filled.png';
+import '../assets/favorite-icon-lined.png';
+
+const restaurantListElement = $('.restaurant-list') as HTMLElement;
+
+export const renderRestaurantList = (restaurantList: RestaurantType[]) => {
+  restaurantListElement.innerHTML = restaurantList
+    .map((restaurant: RestaurantType) => renderRestaurant(restaurant))
```

Comment:
```suggestion
    .map((restaurant) => renderRestaurant(restaurant))
```

이정돈 추론하게 두어도 안전해요~
한번 타입스크립트를 사용할 때 이 부분까지 타입을 명시해줘야하나? 싶으면 마우스를 올려서 타입추론이 제대로 이루어지고 있는지 확인해보세요!
<img width="608" alt="스크린샷 2023-03-05 오후 9 25 31" src="https://user-images.githubusercontent.com/59258239/222960363-417e9a5e-4bef-4fa0-bcc2-826388bcbe1c.png">

Reply:
RestaurantList를 렌더하는 부분은 js로 리팩토링하여 말씀해주신대로 적용하진 못했지만,
다음엔 맵같은 iterable한 메서드들의 원소들은 타입 추론을 하게 두고 확인해보겠습니다!